cachalot/tests/read.py

```diff
@@ -530,23 +540,40 @@ def test_prefetch_related(self):
         self.assertListEqual(permissions8, permissions7)
         self.assertListEqual(permissions8, self.group__permissions)
 
+    def test_test_parent(self):
+        child = TestChild.objects.create(name='child')
+        qs = TestChild.objects.filter(name='child')
+        self.assert_query_cached(qs)
+
+        parent = TestParent.objects.all().first()
+        parent.name = 'another name'
+        parent.save()
+
+        child = TestChild.objects.all().first()
+        self.assertEqual(child.name, 'another name')
```

Comment:
I think we should leave a comment in `models.py` (at least for me) that there's a OneToOneField that was auto created. That's the reason I was confused in our previous convo.